### PR TITLE
CRS-7: Exporting all the missing components and fixing props

### PR DESCRIPTION
**Pull request by vishalnarkhede**
# Submit a pull request

## CLA

- [ ] I have signed the [Stream CLA](https://docs.google.com/forms/d/e/1FAIpQLScFKsKkAJI7mhCr7K9rEIOpqIDThrWxuvxnwUq2XkHyG154vQ/viewform) (required).
- [ ] The code changes follow best practices
- [ ] Code changes are tested (add some information if not applicable)

## Description of the pull request

6a00e54efa53e0883301b8d08d4586970c

YgdjhAHTChoj8uhrbINlzctXz1SyGeCWEHBY

**==== UPDATE ===**

im-ready-now

### CODE REVIEW

File: src/components/Audio.js [hunk 1]
```diff
@@ -1,6 +1,13 @@
 import * as React from 'react';
+import PropTypes from 'prop-types';
 
 export class Audio extends React.Component {
+  static propTypes = {
+    handleClose: PropTypes.func,
```
Comment (tbarbugli) [inline src/components/Audio.js]: @vishalnarkhede please include docs and describe func signature

File: src/components/CommandItem.js [hunk 2]
```diff
@@ -1,6 +1,15 @@
 import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
 
 export class CommandItem extends PureComponent {
+  static propTypes = {
+    entity: PropTypes.shape({
```
Comment (tbarbugli) [inline src/components/CommandItem.js]: @vishalnarkhede jsdoc comment?

File: src/components/EmoticonItem.js [hunk 3]
```diff
@@ -1,6 +1,15 @@
 import React, { PureComponent } from 'react';
+import PropTypes from 'prop-types';
 
 export class EmoticonItem extends PureComponent {
+  static propTypes = {
```
Comment (tbarbugli) [inline src/components/EmoticonItem.js]: @vishalnarkhede docs?

File: src/components/ChannelListTeam.js [hunk 4]
```diff
@@ -36,10 +41,7 @@ class ChannelListTeam extends PureComponent {
           {showSidebar && (
             <div className="str-chat__channel-list-team__sidebar">
               <div className="str-chat__channel-list-team__sidebar--top">
-                <Avatar
-                  image="https://cdn.dribbble.com/users/610788/screenshots/5157282/spacex.png"
-                  size={50}
-                />
+                <Avatar image={this.props.sidebarImage} size={50} />
```
Comment (vishalnarkhede) [inline src/components/ChannelListTeam.js]: @jaapbakker88 this was a leftover thingy I suppose. So replacing it with prop!!

File: src/components/ChannelPreviewLastMessage.js [hunk 5]
```diff
@@ -37,7 +46,7 @@ export class ChannelPreviewLastMessage extends PureComponent {
 
   render() {
     const unreadClass =
-      this.props.unread_count >= 1 ? 'str-chat__channel-preview--unread' : '';
```
Comment (vishalnarkhede) [inline src/components/ChannelPreviewLastMessage.js]: @jaapbakker88 please take a look at this. I tested this and it works fine now, but need second pair eyes!

File: src/components/CommandItem.js (hunk 2 above)
Comment (vishalnarkhede) [reply src/components/CommandItem.js]: Done!

File: src/components/EmoticonItem.js (hunk 3 above)
Comment (vishalnarkhede) [reply src/components/EmoticonItem.js]: done!

File: src/components/Audio.js (hunk 1 above)
Comment (vishalnarkhede) [reply src/components/Audio.js]: Not required anymore!!

**Pull request merged**